src/imports.rs

```diff
@@ -1077,15 +1077,27 @@ impl Rewrite for UseTree {
     fn rewrite(&self, context: &RewriteContext<'_>, mut shape: Shape) -> Option<String> {
         let mut result = String::with_capacity(256);
         let mut iter = self.path.iter().peekable();
+        let start_shape = shape;
+
         while let Some(segment) = iter.next() {
             let segment_str = segment.rewrite(context, shape)?;
+            let mut added_len = first_line_width(&segment_str);
+            if iter.peek().is_some() {
+                added_len += 2; // 3 == "::"
```

Comment:
show this be?
```suggestion
                added_len += 2; // 2 == "::"
```

Reply:
Changed (the 3 was because originally I also added `{`)